src/components/PositionsTable/PositionsTable.js

```diff
@@ -1,76 +1,31 @@
-import React from 'react'
+import React, { memo } from 'react'
 import PropTypes from 'prop-types'
-import _isEqual from 'lodash/isEqual'
+import { VirtualTable } from '@ufx-ui/core'
 import PositionsTableColumns from './PositionsTable.columns'
 
-import Table from '../../ui/Table'
 import './style.css'
 
-class PositionsTable extends React.Component {
-  state = {
-    filteredPositions: [],
-  }
-
-  componentDidMount() {
-    this.getFilteredPositions()
-  }
-
-  shouldComponentUpdate(nextProps, nextState) {
-    return (!_isEqual(nextProps, this.props) || !_isEqual(nextState, this.state))
-  }
-
-  componentDidUpdate() {}
-  getSnapshotBeforeUpdate() {
-    this.getFilteredPositions()
-    return null
-  }
-
-  getFilteredPositions() {
-    const {
-      activeMarket, positions, setFilteredValueWithKey,
-    } = this.props
-    const { marketFilterActive } = this.state
-    const filteredPositions = marketFilterActive
-      ? positions.filter(p => p.symbol === activeMarket.wsID)
-      : positions
-
-    this.setState({ filteredPositions })
-    setFilteredValueWithKey('filteredPositions', filteredPositions)
-    return filteredPositions
-  }
-
-  render() {
-    const { closePosition, authToken } = this.props
-    const { filteredPositions = [] } = this.state
-
-    return (
-      <Table
-        data={filteredPositions}
-        columns={PositionsTableColumns({ authToken, closePosition })}
-        defaultSortBy='mts'
-        defaultSortDirection='ASC'
-      />
-    )
-  }
+const PositionsTable = (props) => {
+  const { closePosition, authToken, positions } = props
+
+  return (
+    <VirtualTable
+      data={positions}
+      columns={PositionsTableColumns({ authToken, closePosition })}
+      defaultSortBy='mts'
```

Comment:
just noticed that we have no 'mts' key in columns, maybe we should use 'cid' instead?
```suggestion
      defaultSortBy='cid'
```